dockerfiles/init/manifests/che.pp

```diff
@@ -40,6 +40,8 @@
   #
   $che_single_port = getValue("CHE_SINGLE_PORT","false")
   $che_single_port_wildcard_domain_host = getValue("CHE_SINGLEPORT_WILDCARD__DOMAIN_HOST","nip.io")
+  $che_single_port_custom_dns = getValue("CHE_SINGLEPORT_CUSTOM_DNS","false")
```

Comment:
Property name doesnt looks self-explained to me.. They're all is a DNS. The only difference is it requires explicit IP or not. So i propose something like `CHE_SINGLEPORT_WILDCARD__DOMAIN_IPLESS`

Reply:
good suggestion!

Reply:
@benoitf i tried to do it like you described, but i am not sure how to elegantly get the configuration into the `SinglePortHostnameBuilder` (without passing it as an argument to the  `SinglePortHostnameBuilder` each time), so I opted for the easier method.